src/main/kotlin/snyk/common/intentionactions/AlwaysAvailableReplacementIntentionAction.kt

```diff
@@ -65,7 +48,7 @@ class AlwaysAvailableReplacementIntentionAction(
     }
 
     companion object {
-        private const val intentionDefaultTextPrefix = "Change to "
-        private const val intentionDefaultFamilyName = "Snyk"
+        private const val intentionTextPrefix = "Upgrade to "
+        private const val intentionTextPostfix = " (Snyk)"
```

Comment:
Hm... I find it easier to have `Snyk` as a prefix, but I suppose that's from UX?

Reply:
Yep, it's UX. Andy suggested to place it like that for readability/clarity